operations/network/management_network/generate_switch_configs.md

```diff
@@ -6,11 +6,8 @@ For example, if there is a suspected configuration issue on single switch, a con
 
 ## Prerequisites
 
-* CANU installed with version 1.1.11 or greater.
-  * Run `canu --version` to see version.
+* CANU installed.
   * If doing a CSM install or upgrade, a CANU RPM is located in the release tarball. For more information, refer to the [Update CANU From CSM Tarball](canu/update_canu_from_csm_tarball.md) procedure.
```

Comment:
probably overlap w/ Jouni's.  Check this shouldn't ref tarball directly, but the release.